aircan/dependencies/google_cloud/bigquery_handler.py

```diff
@@ -31,9 +31,23 @@ def bq_import_csv(table_id, gcs_path, table_schema):
     
 def bq_schema_from_table_schema(table_schema):
     mapping = {
-        'number': 'float',
-        'year': 'integer'
-        }
+        'string': 'string',
+        "number": 'numeric',
```

Comment:
@sagargg you should use single quotes everywhere.

Reply:
updated